src/main/webapp/intrigue-api/graphql.js

```diff
@@ -362,6 +368,16 @@ const metacardTypes = async () => {
     return Object.assign(types, json[group])
   }, {})
 
+  await Promise.all(
```

Comment:
With this addition, I noticed a TTFB increase of about 4x with base DDF. :question: Does that indicate enums should have their own resolver?